### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Cifuuuu**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by Cifuuuu — COMMENTED** [review]
I think the review is good

File: .github/steps/-step.txt [hunk 1]
```diff
@@ -1 +1 @@
-0
+4
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
45
```

File: .github/steps/-step.txt [hunk 2]
```diff
@@ -1 +1 @@
-0
+5
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
6000
```

File: index.html [hunk 3]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (incnotfun) [inline index.html:50]:
```suggestion
READY TO GO
```

File: .github/steps/-step.txt [hunk 4]
```diff
@@ -1 +1 @@
-5
+6000
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:1]:
```suggestion
6000
hello
```

File: .github/steps/-step.txt [hunk 5]
```diff
@@ -1 +1,2 @@
...
+hello
```
Comment (Cifuuuu) [inline .github/steps/-step.txt:2]:
```suggestion

```

**Pull request merged**